final_prep/README.md

```diff
@@ -23,54 +23,55 @@ Some things to consider as you refactor include...
+- [X] I have refactored my `section3` exercises to the best of my ability
 
 ## Gearing Up for Gear Ups
 
 Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry. 
 
-- [ ] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
-  - [ ] Add a link for your gist here: 
-
+- [X] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
+  - [X] Add a link for your gist here: 
+https://gist.github.com/pdthomson/a0ad6eacb394d7d8f580c12ce58e618d
 ## Time Management Prep
 
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
 
-- [ ] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
-- [ ] Study/Project work time (usually about 25 hours/week outside of class time)
-- [ ] Personal, Health + Wellness
+- [X] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
+- [X] Study/Project work time (usually about 25 hours/week outside of class time)
+- [X] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="1440" alt="Screen Shot 2022-02-13 at 1 05 42 AM" src="https://user-images.githubusercontent.com/90064385/153741568-315619b0-0086-4ea6-b747-8ff651609b7f.png">
+
+- <img width="1440" alt="Screen Shot 2022-02-13 at 1 06 01 AM" src="https://user-images.githubusercontent.com/90064385/153742269-d8eb8227-6959-4a41-801c-827050a1eb96.png">`
+
+- <img width="1440" alt="Screen Shot 2022-02-13 at 1 06 08 AM" src="https://user-images.githubusercontent.com/90064385/153742333-22847035-74fc-4413-93ba-f2c4e51d90ee.png">`
+
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
```

Comment:
Your mentorship prep looks pretty good. I would consider adding something in your initial message that addresses what you hope to get out of the mentor relationship. Do you want someone who can review concepts from class with you? Do you want someone to push you? Think about what you might need or want from a mentor and directly ask for it! That will help you land a mentor who is a good fit for you.